tsconfig.json

```diff
@@ -0,0 +1,18 @@
+{
+  "compilerOptions": {
+    /* Language and Environment */
+    "target": "es2016" /* Set the JavaScript language version for emitted JavaScript and include compatible library declarations. */,
+    /* Modules */
+    "module": "CommonJS" /* Specify what module code is generated. */,
+    "typeRoots": [
+      "./node_modules/@types",
+      "./types"
+    ] /* Specify multiple folders that act like './node_modules/@types'. */,
+    /* Interop Constraints */
+    "esModuleInterop": true /* Emit additional JavaScript to ease support for importing CommonJS modules. This enables 'allowSyntheticDefaultImports' for type compatibility. */,
```

Comment:
seems unused

Reply:
Yes you are right. After generating the config file, I copied the configuration from the packages (`image`, `header`) which were already migrated to TypeScript already. I need to change it in `code` package as well.